src/redux/actions/reportPeriod.js

```diff
@@ -8,24 +8,25 @@ export const ACTION_TYPES = {
     REPORT_PERIOD_SELECTED: 'REPORT_PERIOD_SELECTED',
 }
 
+export const errorMessage = i18n.t('Could not load period types')
+
 export const loadPeriodTypesSuccess = periodTypes => ({
     type: ACTION_TYPES.REPORT_PERIOD_TYPES_RECEIVED,
     payload: periodTypes,
 })
 
 export const loadPeriodTypesError = () => ({
     type: ACTION_TYPES.REPORT_PERIOD_TYPES_ERRORED,
-    payload: i18n.t('Could not load period types'),
+    payload: errorMessage,
 })
 
-export const loadPeriodTypes = () => dispatch => {
+export const loadPeriodTypes = () => dispatch =>
     getPeriodTypes()
         .then(periodTypes => dispatch(loadPeriodTypesSuccess(periodTypes)))
         .catch(error => {
             console.error(error)
             dispatch(loadPeriodTypesError())
```

Comment:
console.logs shouldn't make it into master. better use the method mentioned above

Reply:
Oops, my eyes tricked me, saw console.log instead of console.error...